lib/processAllArtifacts.js

```diff
@@ -1,27 +1,64 @@
+var chalk = require('chalk')
 var util = require('util')
 var fs = require('fs')
 var path = require('path')
+
 var readAllNetworkConfigs = require('./readAllNetworkConfigs')
+var updateArtifact = require('./updateArtifact')
+
+var mkdirp = require('mkdirp')
+var exists = util.promisify(fs.exists)
 var readFile = util.promisify(fs.readFile)
 var writeFile = util.promisify(fs.writeFile)
-var updateArtifact = require('./updateArtifact')
 var readdir = util.promisify(fs.readdir)
 
-module.exports = async function (artifactsPath, outputPath) {
-  var networkConfigs = await readAllNetworkConfigs()
-  await readdir(artifactsPath).then(async (filenames) => {
-    await Promise.all(filenames.map((filename) => {
-      var filepath = path.join(artifactsPath, filename)
-      return readFile(filepath).then(json => {
-        var artifact = JSON.parse(json)
-        return updateArtifact(artifact, networkConfigs).then(newArtifact => {
-          var artifactOutputPath = path.join(outputPath, filename)
-          var outputJSON = JSON.stringify(newArtifact, null, 2)
-          return writeFile(artifactOutputPath, outputJSON).then(() => {
-            console.log(`Updated artifact ${filename} to ${artifactOutputPath}`)
+module.exports = async function (inputArtifactsPath, outputArtifactsPath, networksPath) {
+  let inputDirError = false
+
+  const networkConfigs = await readAllNetworkConfigs(networksPath)
+
+  if (!(await exists(inputArtifactsPath))) {
+    console.log(chalk.red(`Input path: ${inputArtifactsPath} does not exist!`))
+    inputDirError = true
+  }
+
+  console.log('outputArtifactsPath', outputArtifactsPath)
+  if (!(await exists(outputArtifactsPath))) {
+    console.log(chalk.green(`Output path: ${outputArtifactsPath} does not exist. Creating...`))
+    mkdirp.sync(outputArtifactsPath)
+  } else {
+    console.log("ALREADY EXISTS")
```

Comment:
I DON'T THINK WE NEED THIS

Heheh I just tried it on the command line and it looks funny